_ru/tour/basics.md

```diff
@@ -26,187 +26,306 @@ previous-page: tour-of-scala
 
 Вы можете комбинировать выражения, окружая их `{}`. Мы называем это блоком.
 
 Результат последнего выражения в блоке будет результатом всего блока в целом.
 
+{% tabs blocks %}
+{% tab 'Scala 2 и 3' for=blocks %}
 ```scala mdoc
 println({
   val x = 1 + 1
   x + 1
 }) // 3
 ```
+{% endtab %}
+{% endtabs %}
 
 ## Функции
 
 Функции — это выражения, которые принимают параметры.
 
 Вы можете определить анонимную функцию (т.е. без имени), которая возвращает переданное число, прибавив к нему единицу:
 
+{% tabs anonymous-function %}
+{% tab 'Scala 2 и 3' for=anonymous-function %}
 ```scala mdoc
 (x: Int) => x + 1
 ```
+{% endtab %}
+{% endtabs %}
 
 Слева от `=>` находится список параметров. Справа — выражение, связанное с параметрами.
 
 Вы также можете назвать функции.
 
+{% tabs named-function %}
+{% tab 'Scala 2 и 3' for=named-function %}
 ```scala mdoc
 val addOne = (x: Int) => x + 1
 println(addOne(1)) // 2
 ```
+{% endtab %}
+{% endtabs %}
 
 Функции могут принимать множество параметров.
 
+{% tabs multiple-parameters %}
+{% tab 'Scala 2 и 3' for=multiple-parameters %}
 ```scala mdoc
 val add = (x: Int, y: Int) => x + y
 println(add(1, 2)) // 3
 ```
+{% endtab %}
+{% endtabs %}
 
 Или вообще не принимать никаких параметров.
 
+{% tabs no-parameters %}
+{% tab 'Scala 2 и 3' for=no-parameters %}
 ```scala mdoc
 val getTheAnswer = () => 42
 println(getTheAnswer()) // 42
 ```
+{% endtab %}
+{% endtabs %}
 
 ## Методы
 
 Методы выглядят и ведут себя очень похоже на функции, но между ними есть несколько принципиальных различий.
 
 Методы задаются ключевым словом `def`.  За `def` следует имя, список параметров, возвращаемый тип и тело.
 
+{% tabs method %}
+{% tab 'Scala 2 и 3' for=method %}
 ```scala mdoc:nest
 def add(x: Int, y: Int): Int = x + y
 println(add(1, 2)) // 3
 ```
+{% endtab %}
+{% endtabs %}
 
 Обратите внимание, как объявлен возвращаемый тип сразу _после_ списка параметров и двоеточия `: Int`.
 
 Методы могут принимать несколько списков параметров.
 
+{% tabs multiple-parameter-lists %}
+{% tab 'Scala 2 и 3' for=multiple-parameter-lists %}
 ```scala mdoc
 def addThenMultiply(x: Int, y: Int)(multiplier: Int): Int = (x + y) * multiplier
 println(addThenMultiply(1, 2)(3)) // 9
 ```
+{% endtab %}
+{% endtabs %}
 
 Или вообще ни одного списка параметров.
 
+{% tabs no-parameter-lists %}
+{% tab 'Scala 2 и 3' for=no-parameter-lists %}
 ```scala mdoc
 def name: String = System.getProperty("user.name")
 println("Hello, " + name + "!")
 ```
+{% endtab %}
+{% endtabs %}
 
 Есть некоторые отличия от функций, но пока что их можно рассматривать как нечто похожее.
 
 Методы также могут иметь многострочные выражения.
 
+
+{% tabs get-square-string class=tabs-scala-version %}
+
+{% tab 'Scala 2' for=get-square-string %}
 ```scala mdoc
 def getSquareString(input: Double): String = {
   val square = input * input
   square.toString
 }
 println(getSquareString(2.5)) // 6.25
 ```
+{% endtab %}
+
+{% tab 'Scala 3' for=get-square-string %}
+```scala
```

Comment:
Maybe additional `mdoc`? i've checked this variant (it's works)

```suggestion
```scala mdoc
```

Reply:
Unfortunately, this is not possible because this section of the documentation (`"Tour of Scala"`) compiles under 2.13.

I already tried to do this, but without success:
https://github.com/scala/docs.scala-lang/pull/2780/checks?check_run_id=13144429094

We can do this only for `scala3-book` section.